src/main/java/org/jsoup/select/NodeTraversor.java

```diff
@@ -1,32 +1,54 @@
 package org.jsoup.select;
 
+import org.jsoup.helper.Validate;
+import org.jsoup.nodes.Element;
 import org.jsoup.nodes.Node;
+import org.jsoup.select.NodeFilter.FilterResult;
 
 /**
- * Depth-first node traversor. Use to iterate through all nodes under and including the specified root node.
+ * Depth-first node traversor. Use to iterate through all nodes under and
```

Comment:
Reformatting this JavaDoc comment and some others below seems unnecessary. Probably caused by your IDEs automatic code formatting.